packages/utils/src/constant.js

```diff
@@ -4,5 +4,13 @@
  */
 
 export const VALUE_TYPES = {
+  BOOLEAN: 'boolean',
```

Comment:
I'd suggest to move this constant in `formatDataValueByType.js` and export it from there (I know you didn't add this file, but its existence doesn't make sense to me)

Reply:
Yes good suggestion!